x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/field_parameters/inference_id_selects.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiCallOut } from '@elastic/eui';
+import React, { useEffect, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+
+import { useComponentTemplatesContext } from '../../../../component_templates/component_templates_context';
+import { getFieldConfig } from '../../../lib';
+import {
+  FieldHook,
+  Form,
+  FormDataProvider,
+  SuperSelectField,
+  UseField,
+  useForm,
+} from '../../../shared_imports';
+import { SuperSelectOption } from '../../../types';
+interface Props {
+  onChange(value: unknown): void;
+  'data-test-subj'?: string;
+}
+
+export const InferenceIdSelects = ({ onChange, 'data-test-subj': dataTestSubj }: Props) => {
+  const { form } = useForm({ defaultValue: { main: 'elser' } });
+  const { subscribe } = form;
+  const { api } = useComponentTemplatesContext();
+  const [inferenceModels, setInferenceModels] = useState<any>([]);
```

Comment:
question:  are we defaulting to ELSER v2 model?

Reply:
Yes, I have committed the changes to accommodate this. Therefore, the default inference_id should be 'elser_model_2'. If 'elser_model_2' does not exist as an inference endpoint, it will be created in the background using the inference API. The same process applies for 'e5'. In the forthcoming PR, I will implement changes so that whenever the user clicks the 'Add field' button, the inference_ids for 'elser_model_2' and 'e5' will be automatically created in the background if they do not exist.

```
PUT _inference/sparse_embedding/elser_model_2
{
  "service": "elser",
  "service_settings": {
    "num_allocations": 1,
    "num_threads": 1
  },
  "task_settings": {}
}
```

Reply:
Now, the default is elser_model_v2